src/pages/spawn/spawnHost/SpawnHostTableActions.tsx

```diff
@@ -14,28 +14,62 @@ const { copyToClipboard } = string;
 export const SpawnHostTableActions: React.FC<{ host: MyHost }> = ({ host }) => (
   <FlexContainer>
     <SpawnHostActionButton host={host} />
-    <CopySSHCommandButton host={host} />
+    <CopySSHCommandButton user={host.user} hostUrl={host.hostUrl} />
     <EditSpawnHostButton host={host} />
   </FlexContainer>
 );
 
-const CopySSHCommandButton: React.FC<{ host: MyHost }> = ({ host }) => {
-  const sshCommand = `ssh ${host.user}@${host.hostUrl}`;
+export const CopySSHCommandButton: React.FC<{
+  user: string;
+  hostUrl: string;
+}> = ({ user, hostUrl }) => {
+  const sshCommand = `ssh ${user}@${hostUrl}`;
   const spawnAnalytics = useSpawnAnalytics();
+  const [hasCopied, setHasCopied] = useState(false);
+  const [openTooltip, setOpenTooltip] = useState(false);
+  useEffect(() => {
+    const timeout = setTimeout(() => setHasCopied(false), 10000);
```

Comment:
nit. Can you use the SECOND constant from https://github.com/evergreen-ci/spruce/blob/6cf75a90683e6bb58e5bccd9ebce41f4683e9675/src/constants/index.ts

It makes it a little more readable to see `10 * SECOND` and easily know this is a 10 second time out instead of trying to do the math/ count the 0's

Reply:
done